xml/depl_require.xml

```diff
@@ -1406,6 +1406,110 @@
     For storage requirements, see <xref linkend="sec.depl.req.storage.hardware.store"/>.
    </para>
   </sect2>
+
+<sect2 xml:id="sec.depl.req.hardware.monnode">
+   <title>&monnode;</title>
+   <para>
+     The Monasca agent roles, such as <literal>monasca-agent</literal> and
+     <literal>monasca-log-agent</literal>, should be assigned to all nodes,
+     regardless of their other roles. However, the
+     <literal>monasca-server</literal> role must be deployed on a dedicated physical machine
+     referred as &monnode;. Hardware required for the &monnode; are as follows:
+   </para>
+   <itemizedlist mark="bullet" spacing="normal">
+    <listitem>
+     <para>
+       Architecture: x86_64
+     </para>
+     </listitem>
+    <listitem>
+     <para>
+       RAM: At least 32 GB, 64 GB or more is recommended
+     </para>
+     </listitem>
+    <listitem>
+     <para>
+       CPU: At least 8 cores, 16 cores or more is recommended
+     </para>
+     </listitem>
+    <listitem>
+     <para>
+       Hard Disk: SSD is strongly recommended
+     </para>
+     </listitem>
+   </itemizedlist>
+   <para>
+     The following formula can be used to calculate the required disk space:
+   </para>
+   <screen>200 GB + ["number of nodes * "retention period" * ("space for log
+   data/day" + "space for metrics data/day") ]</screen>
+   <para>
+       The recommended values for the formula are as follows:
+   </para>
+    <itemizedlist mark="bullet" spacing="normal">
+    <listitem>
+     <para>
+       Retention period = 60 days for InfluxDB and Elasticsearch
+     </para>
+     </listitem>
+    <listitem>
+     <para>
+       Space for daily log data = 2GB
+     </para>
+     </listitem>
+    <listitem>
+     <para>
+       Space for daily metrics data = 50MB
+     </para>
+     </listitem>
+   </itemizedlist>
+   <para>
+     The formula is based on the following log data assumptions:
+   </para>
+   <itemizedlist mark="bullet" spacing="normal">
+    <listitem>
+     <para>
+       Approximately 50 log files per node
+     </para>
+     </listitem>
+    <listitem>
+     <para>
+       Approximately 1 log entry per file per sec
+     </para>
+     </listitem>
+    <listitem>
+     <para>
+       200 bytes in size
+     </para>
+     </listitem>
+   </itemizedlist>
+   <para>
+     The formula is based on the following metrics data assumptions:
+   </para>
+   <itemizedlist mark="bullet" spacing="normal">
+   <listitem>
+     <para>
+       400 metrics per node
+     </para>
+     </listitem>
+   <listitem>
+     <para>
+      Time interval of 30 seconds
+     </para>
+     </listitem>
+   <listitem>
+     <para>
+      20 bytes in size
+     </para>
+     </listitem>
+   </itemizedlist>
+   <para>
+     Please note that the formula provides only a rough estimation of the
```

Comment:
Sounds sensible, but I think the assumptions and recommended values take precedence over the note. In other words, it's more important to clarify the formula's values and basis before providing additional info. Thoughts?